fhir-search/src/test/java/com/ibm/fhir/search/parameters/ParametersUtilTest.java

```diff
@@ -40,10 +40,10 @@
 import com.ibm.fhir.search.test.BaseSearchTest;
 
 /**
- *
  * Tests ParametersUtil
  */
 public class ParametersUtilTest extends BaseSearchTest {
+    public static final boolean DEBUG = true;
```

Comment:
turn it off?

Reply:
done (and also in ParametersSearchUtilTest)